.github/gobench.yml

```diff
@@ -0,0 +1,9 @@
+checks:
+- name: max NS per Op
+  description: |-
+    % difference for NsPerOp in the diff function
+  package: .
+  benchmarks: []
```

Comment:
I don't believe it's the empty benchmarks file - see https://github.com/bobheadxi/gobenchdata/commit/99aee361f4e5333ab9cc3b92c1628b517edd022c

I suspect it might be this, looking into it

Reply:
Ah nevermind, it is the empty benchmarks file - see https://sourcegraph.com/github.com/bobheadxi/gobenchdata/-/blob/checks/evaluate.go?utm_source=VSCode-1.1.0#L100:31